Scripts/Source/UD_AbadonPlug_RenderScript.psc

```diff
@@ -188,18 +188,18 @@ Function randomEquipHandRestrain()
                 debug.messagebox("Suddenly, Abadon plug starts to emit black smoke. Before you could react, smoke forces your hands away from your body and locks them in to cold, steel yoke.")
             endif
         else
-            UDCDmain.Print(getDeviceName() + " locks restrain on " + getWearerName())
+            UDCDmain.Print(getDeviceName() + " locks restraint on " + getWearerName())
         endif
     endif
 EndFunction
 
 bool Function equipRandomRestrain()
-    Armor device = UDmain.UDRRM.getRandomSuitableRestrain(getWearer(),0xFBFB)
+    Armor device = UDmain.UDRRM.getRandomSuitableRestrain(getWearer(),0xFFFCFFF3) ; exclude belts and plugs
```

Comment:
Either update the UDRRM or undo the changes (see comment in UDRRM)

Reply:
Yep.

Reply:
Done.